pages/_app.tsx

```diff
@@ -3,6 +3,12 @@ import { AppProps } from 'next/app'
 import { ReactElement } from 'react'
 import { appWithTranslation } from 'next-i18next'
 
+import { config } from 'dotenv'
+import { resolve } from 'path'
+
+// TODO: Make this path resolution more robust (no relative paths!)
+config({path: resolve(process.cwd(), '..','.env')})
```

Comment:
mind adding a comment about why this is necessary? i assume it was working before and this makes it work for some additional case?...

Reply:
Although we had an existing `.env` filed, it was solely being used for Storybook purposes, and that software looks for a .env file.  In our case, we want to use the environmental variables in app code, so dotenv is being used to expose them on the node's global `process` variable (the `process.env` object, to be specific).  

https://www.chromatic.com/docs/cli

Comment updated!